profiles/apparmor/template.go

```diff
@@ -17,6 +17,10 @@ profile {{.Name}} flags=(attach_disconnected,mediate_deleted) {
   capability,
   file,
   umount,
+{{if ge .Version 208096}}
```

Comment:
It looks like signal mediation functionality appears in apparmor 2.9.0, 
as per https://gitlab.com/apparmor/apparmor/wikis/Release_Notes_2.9.0.

Oh, OK, I see that there was a bug in Ubuntu shipping 2.9.0 as 2.8.95. Maybe makes sense to add a comment?

Reply:
still unclear why you have it set to 2.8.96 @cyphar

Reply:
It's because the Ubuntu AppArmor versions are slightly different -- from memory 2.8.95 *does not* have the change but anything after it does. I can double check this though (finding the versions requires you to trawl through the different branches on Launchpad -- the official release notes don't tell the full story for some reason).

Reply:
OK, I guess it's enough to share the above info in the patch commit message.